src/Stripe.net/Entities/Subscriptions/Subscription.cs

```diff
@@ -229,20 +301,62 @@ public SetupIntent PendingSetupIntent
         [JsonConverter(typeof(DateTimeConverter))]
         public DateTime? StartDate { get; set; }
 
+        /// <summary>
+        /// Possible values are <c>incomplete</c>, <c>incomplete_expired</c>, <c>trialing</c>,
+        /// <c>active</c>, <c>past_due</c>, <c>canceled</c>, or <c>unpaid</c>.
+        /// </summary>
         [JsonProperty("status")]
         public string Status { get; set; }
 
+        #region Expandable Schedule
```

Comment:
The entire region should be after `Quantity` and before `Start`.

Reply:
Damn it, I had named it SubscriptionSchedule at first :p